docs/app/components/component-section/component-section.component.html

```diff
@@ -12,19 +12,58 @@ <h3 class="uxd-component-section-title">{{ title }}</h3>
             <uxd-usage-link *ngIf="usage" [usage]="usage"></uxd-usage-link>
 
             <!-- Show the Angular Version -->
-            <span class="label label-primary uxd-component-section-labels" *ngIf="version === 'AngularJS'">AngularJS</span>
-            <span class="label label-accent uxd-component-section-labels" *ngIf="version === 'Angular'">Angular</span>
+            <span *ngIf="version === 'AngularJS'" class="label label-primary uxd-component-section-labels">AngularJS</span>
+            <span *ngIf="version === 'Angular'" class="label label-accent uxd-component-section-labels">Angular</span>
+
+            <a *ngIf="hybrid" href="javascript:void(0)" class="label label-accent uxd-component-section-labels"
+                [popover]="hybridHelpContent" container="body" placement="bottom" [outsideClick]="true"
+                containerClass="uxd-component-section-help-popover">
+                Angular Compatible
+            </a>
 
             <!-- Identify as External plugin -->
-            <a class="label label-warning uxd-component-section-labels" tooltip="This component is from a third party library and not included in the UX Aspects component library." container="body" *ngIf="externalUrl" [href]="externalUrl" target="_blank">External</a>
+            <a *ngIf="externalUrl" role="button" class="label label-warning uxd-component-section-labels"
+                tooltip="This component is from a third party library and not included in the UX Aspects component library."
+                container="body" placement="bottom" [href]="externalUrl" target="_blank">
+                External
+            </a>
 
             <!-- Show Deprecated label if required -->
-            <span class="label label-critical uxd-component-section-labels" tooltip="This component has been deprecated and will be removed in the next major version of UX Aspects." container="body" *ngIf="deprecated">Deprecated</span>
+            <span *ngIf="deprecated" class="label label-critical uxd-component-section-labels"
+                tooltip="This component has been deprecated and will be removed in the next major version of UX Aspects."
+                container="body" placement="bottom">
+                Deprecated
+            </span>
         </div>
 
     </ux-ebox-header>
 
     <ux-ebox-content class="uxd-component-section-content">
         <div #container></div>
     </ux-ebox-content>
-</ux-ebox>
\ No newline at end of file
+</ux-ebox>
+
+<ng-template #hybridHelpContent>
+    <p>
+        To use this AngularJS component in an Angular 5 application, import <code>HybridModule</code> into the module
```

Comment:
Possibly removed the 5 from Angular 5 so we don't have to keep updating this?

Reply:
Yeah true. Would the hybrid components work in a site based on Angular 4 or lower?

Reply:
They should work fine in an Angular 4 or 5 application. I don't believe they will work in Angular 2 applications, however Angular releases are only designed to be backwards compatible with the previous major release so we can only ever support the 2 most recent major versions of Angular anyway so none of our library is guaranteed to work with Angular 2 anymore.